spec/fixtures/shared_dict.lua

```diff
@@ -13,6 +13,8 @@ local dicts = {
   "kong_db_cache_2 16m",
   "kong_db_cache_miss 12m",
   "kong_db_cache_miss_2 12m",
+  "kong_dns_cache 10m",
+  "kong_dns_cache_miss 10m",
```

Comment:
Should be `12m` and `5m`?

Reply:
yes, these value in test cases should be algined with kong configuration. Another work here I need to do is to test how much shared memory can accommodate the default size of DNS record data.